src/web/assets/js/photoGallery.js

```diff
@@ -0,0 +1,102 @@
+            ""
+        ],[
+            "https://lh3.googleusercontent.com/BjHxy1hbfuk_Pz1IG90W62hfx8dMwetLBM2XWdyZiLVRbwjlkReX56GHOUzlShS69oljefufyVRcAdKW2R_FhU8LeKjYHtJl-chS6fwOIdzeQxgjcyJR9kXDi75qBo0c1rqf6emuN5SQVEwkUQO-qAYhSX-myqZis4lGtwxUv3e5fvsLbKGC9OX1eTiJD8epLNelSQ1m8ptkEkaLc2vpSeOsh9E0O2TYc2Z2IEqRbIbT1kG0poe_dLactwcE6OiGRjzHMmaVqG8KbqMQoA1cLuUa1EhMyNquAjS6jhsLD_wkblfz47IvFQbibBp1eiFHBcsuhrcga9ciVFfKsNKRyCcmAVMDO6Ybrh6f_nZoCKfBgcj1OOV1mbcAvypPZm4ptN5lvqA1__mbilPTuq5pJCRhYGraVjsXdiBS7pwr6rsyVVS8eDQraECmovlJimVE1IKq0a5VWUaiUx3Vk7kSRQoXiww1x3s56-GVB3s3OvoArVxISFu6SCrb8pL1dfCVHbBEWgKv5O-aQqV7iGyrwVq3NFRMrZs-UGoKhR2gxyt_z32ZfOQNjNG42b6Ct9fIFkBBGoPC0TUHKRz7djdyIN_NBxOhBkQ=w519-h778-no",
+            ""
+        ],[
+            "https://lh3.googleusercontent.com/CdBajYQ_Cjzd3THX_ZDV_C_x9B9iftaAds5hJQ-7o77wCyBYS1tdjYWfY8cHUth9-ajT5CxW0_aaFhanmY4flS0fHx0Wfxybp0tcIO7-NH5eVOHo_frBFNHA1OmJVPZieSTj6eCqx1DWlQZkTpeo6trB_q3zPoCY74PTiLYdCea1b8zJ5wlOkiGktM-5r20TwtjugdXG4t0bEDjNfPwrood2ftnPzjBJ-SrYJDtneQzGdtAk2I2zks7N-EPO_iA39I3NXdMa0zv3D3PlJNo_6YqYMOYNvDuDaP8s41S13AsXOfsk2cDHg_5nzoYy_4p7lIos16PbArlpJKoEPlq19MvKZ-uQzH69ycmQV6EL70v2Om2OLWhQojGETSQ9AFW6bh7p8IgJHrUsUW_xy9cDhwYsAbGnGmu3rEb_6RbnEei-zoAmkdFtbbdE_JCgX5NShm8aMbdL2UvOjholdPFHmzeBrpjC6llcaCGOIwxYtIdQjX6Zts6W4XIm5fIEe-9Wbi2LHdq_ARO44Y0eueLmLacIJ1Z-PHBb9_jp1VNIFBXpl5h5q7gAwMpcf5JctMKfuSF-q3tUofebFy_-xqhHcU9g2BDoP-4=w519-h778-no",
+            ""
+        ],
+    ];
+    var imageIndex = 0; //tracks current index of displayed photos when inserting each image into DOM
+
+    function addPhotoToGallery(_this,hiddenDiv) { //add photos to THIS gallery
+        var imageSrc = photos[imageIndex][0];
+        var imageTitle = photos[imageIndex][1];
+
+        $(_this).append(" \
+            <div class='" + hiddenDiv + " col-lg-" + (12/numOfPhotos) + " flex'> \
+                <a href='" + imageSrc + "' data-toggle='lightbox' data-type='image' data-gallery='barcampGallery' data-title='" + imageTitle + "'> \
+                    <img src='" + imageSrc + "' alt='" + imageTitle + "' class='img-responsive photoGallery'> \
+                </a> \
+            </div> \
+        ");
+
+        imageIndex++; //imageIndex cycles through photos array
+        if (imageIndex === photos.length){imageIndex = 0} //loop index if it reaches the end and start over
+    }
+
+    $('section:not(".section-pattern") > div').each(function(){ //create a gallery for each section that matches
```

Comment:
This is not too clear. I'd prefer explicitly adding this markup in the template rather than generating it in the javascript. Otherwise it's difficult to tell where this markup will be added since the selector is not unique.